.github/workflows/ci.yml

```diff
@@ -34,13 +34,13 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         browser: [chrome, firefox]
-        node: ['14', '16', '18']
```

Comment:
~~making note of braking change: dropping node 14~~

nevermind, this isn't a breaking change.

ember-page-title is a v2 addon so testing against node doesn't matter.

We could have gotten rid of the whole node matrix here